lib/cisco_node_utils/bgp_neighbor_af.rb

```diff
@@ -70,6 +70,13 @@ def validate_args(asn, vrf, nbr, af)
       fail ArgumentError, "'af' must be an array specifying afi and safi" unless
         af.is_a?(Array) || af.length == 2
 
+      # XR BGP does not support <address>/<mask>
```

Comment:
@glennmatthews Addressed your last round of comments. Rubocop and mini test happy.